typescript/simple-mercari-web/Dockerfile

```diff
@@ -1,7 +1,15 @@
 FROM node:16-alpine
 WORKDIR /app
+USER root
 
-RUN addgroup -S mercari && adduser -S trainee -G mercari
-USER trainee
+COPY package.json package.json
+COPY package-lock.json package-lock.json
+COPY tsconfig.json tsconfig.json
+COPY src src
+COPY public public
+RUN chmod -R 755 /app
+RUN npm ci
+EXPOSE 3000
+#RUN addgroup -S mercari && adduser -S trainee -G mercari
```

Comment:
nit: a better security practice is to run commands as a non-root user in a container.